safenode/src/bin/cli/wallet.rs

```diff
@@ -0,0 +1,148 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use safenode::{
+    client::{Client, WalletClient},
+    protocol::wallet::{parse_public_address, DepositWallet, LocalWallet, Wallet},
+};
+
+use sn_dbc::{Dbc, Token};
+
+use clap::Parser;
+use eyre::Result;
+use std::path::PathBuf;
+use tokio::fs;
+use tracing::{info, warn};
+use walkdir::WalkDir;
+
+#[derive(Parser, Debug)]
+pub enum WalletCmds {
+    Deposit {
+        /// Tries to load a hex encoded `Dbc` from the
+        /// given path and deposit it to the wallet.
+        #[clap(name = "dbc-dir")]
+        dbc_dir: PathBuf,
```

Comment:
Why does this point to a directory rather than a file? Is it going to deposit all the files in the directory?

Reply:
Yep.